CONTRIBUTING.md

```diff
@@ -31,9 +31,9 @@ There are many ways to help us out and we appreciate all of them.
 
 ## <a name="communication"> Communication </a>
 
-### <a name="slack"> Slack Chat </a>
+### <a name="chat"> Chat </a>
 
-The quickest and most open way to communicate with the Redox team is with [Slack](https://slack.com). Currently, the only way to join our slack team is by sending an email to [info@redox-os.org](mailto:info@redox-os.org), which might take a little while, since it's not automated. We're currently working on an easier way to do this, but this is the most convenient way right now.
+The quickest and most open way to communicate with the Redox team is on our [chat server](http://chat.redox-os.org)
```

Comment:
Oh, okay, sorry. I guess I didn't really think about that. I was focused on replacing every occurence of "slack" with simply "the chat". I'll make some minor changes appending this when I get home (I actually did this on my phone)